### ISSUE TITLE
Final performance of the model

### ISSUE THREAD

**Issue by z2230493358** [DJEddyking/LF2#4]
Hello sir, I recently ran your code and did not make any changes to your code. But the mAP of market to duke is only 70.6 and the mAP of duke to market is 80.4. What could be the problem?

Regarding the parameters:
source_pretrain.py    
arch='resnet50_source', batch_size=64, dropout=0, epochs=80, eval_step=5, evaluate=False, features=0, height=256, iters= 200, lr=0.00035, margin=0.0, milestones=[40, 70], momentum=0.9, num_instances=4, seed=1, warmup_step=10, weight_decay=0.0005, width=128, workers=4
target_train.py    
alpha=0.999, arch='resnet50', batch_size=64, dropout=0, epochs=80, eval_step=1, features=0, height=256,iters=400,lr= 0.00035, momentum=0.9, num_clusters=700, num_instances=4, seed=1, split_parts=2, weight_decay=0.0005, width=128, workers=4
model_test.py    
arch='resnet50', batch_size=64,dropout=0, features=0, height=256, num_classes=700,seed=1, split_parts=2, width=128, workers= 4

**Comment by DJEddyking** [comment]
Thank you for your interest in our work. After receiving your question, we used the code from the repository without changing anything anywhere and re-run the duke to market experiment. The weights and logs are at this link: [password: abcd](https://pan.baidu.com/s/1i2oETgkIB9qDPvAuLeO_5Q). You can check if the training parameters are consistent.
We guess the problem you are experiencing may be that the library or package used in the code is not updated to the latest version, such as sklearn, etc. Or maybe it's a code environment configuration issue.

**Comment by z2230493358** [comment]
> Thank you for your reply, I will try again. And thanks for sharing your work!
